rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -637,17 +614,15 @@ type OptionalEarlyTermination: <"A data defining:  an early termination provisio
 
     singlePartyOption BuyerSeller (0..1) <"If optional early termination is not available to both parties then this component specifies the buyer and seller of the option. In FpML, this attribute is of type SinglePsrtyOption, which actually consists of the BuyerSeller.model.">
     mutualEarlyTermination boolean (0..1) <"Used for specifying whether the Mutual Early Termination Right that is detailed in the Master Confirmation will apply.">
-    americanExercise AmericanExercise (0..1) <"American exercise. FpML implementations consists in an exercise substitution group.">
-    bermudaExercise BermudaExercise (0..1) <"Bermuda exercise. FpML implementations consists in an exercise substitution group.">
-    europeanExercise EuropeanExercise (0..1) <"European exercise. FpML implementations consists in an exercise substitution group.">
     exerciseNotice ExerciseNotice (0..*) <"Definition of the party to whom notice of exercise should be given.">
     followUpConfirmation boolean (0..1) <"A flag to indicate whether follow-up confirmation of exercise (written or electronic) is required following telephonic notice by the buyer to the seller or seller's agent.">
     calculationAgent CalculationAgent (0..1) <"The ISDA Calculation Agent responsible for performing duties associated with an optional early termination.">
     cashSettlement SettlementTerms (0..1) <"If specified, this means that cash settlement is applicable to the transaction and defines the parameters associated with the cash settlement procedure. If not specified, then physical settlement is applicable.">
     optionalEarlyTerminationAdjustedDates OptionalEarlyTerminationAdjustedDates (0..1) <"An early termination provision to terminate the trade at fair value where one or both parties have the right to decide on termination.">
+    exerciseTerms ExerciseTerms (1..1)
```

Comment:
Please a brief comment for ```exerciseTerms```, like you have done for the instance under ```OptionPayout```

Reply:
Done - in #2687